CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# POA Network
+
+## To Contribute
+1. Fork the repository
+https://github.com/poanetwork/hbbft
+2. Create a feature branch
+3. Write tests to cover the work 
+4. Commit changes
```

Comment:
Before committing changes the contributor has to ensure the CI rules are satisfied, in particular, formatting is correct and all CI tests pass: https://github.com/poanetwork/hbbft/blob/1436d854554edd879ac9f68ffe60c9a4cf6f3747/.travis.yml#L21
Otherwise they may be waiting for acceptance of a PR that doesn't pass CI tests.

Reply:
Good point! We should definitely link to the `.travis.yml` here, and explain that you have to use the same Rust, Clippy and Rustfmt versions as configured in the file.

Reply:
Thanks I will be sure to add.